tests/test_image.py

```diff
@@ -62,17 +62,22 @@ def test_ImageReader(data_path):
     pipeline.run()
 
 def test_Gray2RGB():
-    image = skimage.data.camera()
+    image = skimage.data.astronaut()
```

Comment:
`camera` is right as you need a grayscale image

Reply:
It raises the error saying that image.shape != 3 whenever I use camera.
![cameraerror](https://user-images.githubusercontent.com/24892256/68401641-64fa8900-017a-11ea-9aea-6a33a306d742.JPG)

Reply:
camera in Gray2RGB, astronaut in RGB2Gray.